mcs/class/System.ServiceModel.Syndication/System.ServiceModel.Syndication.dll.sources

```diff
@@ -1,40 +1,10 @@
 ../../build/common/Consts.cs
 ../../build/common/Locale.cs
 ../../build/common/MonoTODOAttribute.cs
+../../build/common/SR.cs
 Assembly/AssemblyInfo.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Atom10FeedFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Atom10FeedFormatter_1.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Atom10ItemFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Atom10ItemFormatter_1.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/AtomPub10CategoriesDocumentFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/AtomPub10ServiceDocumentFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/AtomPub10ServiceDocumentFormatter_1.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/CategoriesDocument.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/CategoriesDocumentFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/ISyndicationElement.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/InlineCategoriesDocument.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/ReferencedCategoriesDocument.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/ResourceCollectionInfo.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Rss20FeedFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Rss20FeedFormatter_1.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Rss20ItemFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/Rss20ItemFormatter_1.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/ServiceDocument.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/ServiceDocumentFormatter.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/SyndicationCategory.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/SyndicationContent.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/SyndicationElementExtension.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/SyndicationElementExtensionCollection.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/SyndicationExtensions.cs
-../System.ServiceModel.Web/System.ServiceModel.Syndication/SyndicationFeed.cs
```

Comment:
There seems to be _another_ set of these files in `mcs/class/System.ServiceModel.Web/obsoleted/System.ServiceModel.Syndication` which we can remove as well.

Reply:
Done!